integration/tests/trainium/trainium_test.go

```diff
@@ -238,20 +234,24 @@ func getAvailabilityZones(ctx context.Context, ec2API awsapi.EC2) (string, strin
 	Expect(err).NotTo(HaveOccurred())
 	zones := output.AvailabilityZones
 
+	zoneMap := map[string]struct{}{}
```

Comment:
Curious, is map[string]struct{}{} a style thing?  Or is it more efficient than map[string]bool for this case?

Reply:
It's more efficient, takes fewer bytes, and is more idiomatic when you're using a map as a set.

Reply:
Not sure if you saw my message on Slack, but the integration tests are failing in our account.